react/client/src/store/reducers/privacy.duck.js

```diff
@@ -0,0 +1,33 @@
+export const PRIVACY_ACCEPTED_UPDATE = 'truth-loop/privacy/privacyAcceptedUpdate'
+export const PRIVACY_CANCELLED_UPDATE = 'truth-loop/privacy/privacyCancelledUpdate'
+
+export const DEFAULT_STATE = {
+  privacyAccepted: false,
+  privacyCancelled: false,
+}
+
+export default function reducer(state = DEFAULT_STATE, action = {}) {
+  if (action.payload === null || typeof action.payload === 'undefined') {
```

Comment:
I intend to move this into the case statements so we don't override the behavior of all action reducers. I don't think it will hurt us right not though.